docker-compose.yml

```diff
@@ -32,11 +32,19 @@ services:
     volumes:
       - ./.localstack:/docker-entrypoint-initaws.d
 
+  cryptor:
+    build: etc/cryptor-mock
+    container_name: cryptor
+    environment:
+      CRYPTOR_CLIENT_IDENTIFICATION_CHALLENGE: "12345" #has to match the one in env.test
```

Comment:
instead of injecting it here (and duplicating the value) you could get the cryptor mock to use to .env file to get the value